From bf69fe0ed2d5e09cccfcd78062a974a0e761033b Mon Sep 17 00:00:00 2001
From: Dominique Martinet <asmadeus@codewreck.org>
Date: Sat, 23 Apr 2022 23:03:57 +0900
Subject: [PATCH 4/4] tools/bpf: replace sys/fcntl.h by fcntl.h

musl does not like including sys/fcntl.h directly:
    1 | #warning redirecting incorrect #include <sys/fcntl.h> to <fcntl.h>

Signed-off-by: Dominique Martinet <asmadeus@codewreck.org>
---
accepted by upstream without modification, can be dropped after update
to v5.19 or higher (merged in v5.19-rc1 as
246bdfa52f33ecfa52546ed67287de4eab165b2e )

 tools/bpf/bpftool/tracelog.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/bpf/bpftool/tracelog.c b/tools/bpf/bpftool/tracelog.c
index e80a5c79b38f..bf1f02212797 100644
--- a/tools/bpf/bpftool/tracelog.c
+++ b/tools/bpf/bpftool/tracelog.c
@@ -9,7 +9,7 @@
 #include <string.h>
 #include <unistd.h>
 #include <linux/magic.h>
-#include <sys/fcntl.h>
+#include <fcntl.h>
 #include <sys/vfs.h>
 
 #include "main.h"
-- 
2.35.1

